O/openfhe_julia/build_tarballs.jl

```diff
@@ -77,8 +77,8 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    BuildDependency(PackageSpec(;name="libjulia_jll", version=v"1.10.7")),
-    Dependency(PackageSpec(name="libcxxwrap_julia_jll", uuid="3eaa8342-bff7-56a5-9981-c04077f7cee7"); compat="0.11.2"),
+    BuildDependency(PackageSpec(;name="libjulia_jll", version=v"1.10.9")),
+    Dependency(PackageSpec(name="libcxxwrap_julia_jll", uuid="3eaa8342-bff7-56a5-9981-c04077f7cee7"); compat="0.12.2"),
```

Comment:
I presume you're aware this may (or may note) require changes in your CxxWrap code?  Just checking!

Reply:
Thanks for being vigilant though 🙏

Reply:
Thank you very much, yes we are aware!)